cmake/TPLs/FindTPLCUBLAS.cmake

```diff
@@ -59,6 +59,8 @@ ELSE()
   find_library(CUDA_cublas_LIBRARY
     cublas
     HINTS ${CUDA_TOOLKIT_ROOT_DIR}/lib
+          ENV TRILINOS_CUBLAS_ROOT_DIR
```

Comment:
Why is this needed?

Reply:
Originally branched off master and this was resolved differently in develop since then. I'll stick with the current develop approach.